DMX.Core.Api/Models/Configurations/ApiConfigurations.cs

```diff
@@ -6,7 +6,8 @@ namespace DMX.Core.Api.Models.Configurations
 {
     public class ApiConfigurations
     {
-        public string Url { get; set; }
-        public string AccessKey { get; set; }
+        public string Url { get; set; }
+        public string GetAllDevicesAccessKey { get; internal set; }
+        public string GetAvailableDevicesAccessKey { get; internal set; }

```

Comment:
Change these property names from verb to nouns and remove the `internal` part.

Reply:
I changed the property names, and assumed that the CloudBroker.WebApps variable should also be updated. Is that correct?